example/monster/README.md

```diff
@@ -0,0 +1,37 @@
+# The monster demo
```

Comment:
Could you please add a **Get Startted** section to the README, descript how to start your demo bot, like run `run-monster-bot.sh`?